Templates/HttpTrigger-Powershell/run.ps1

```diff
@@ -0,0 +1,8 @@
+$name = Get-Content $Env:req
```

Comment:
How about coding this like the BAT template:

if ($Env:req_query_name) 
{
	$res = "Hello $Env:req_query_name"
}
else
{
	$res = "Please pass a name on the query string"
}

Out-File -NoNewline $Env:res -InputObject $res